config/prometheus/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources:
```

Comment:
Do we want prometheus coming in with this change? This question applies to all the files in the `config/prometheus` directory.